nautobot/docs/user-guides/getting-started/creating-devices.md

```diff
@@ -0,0 +1,97 @@
+# Creating Devices in Nautobot
+
+A network Device in Nautobot has a few required attributes:
+
+* A Device Role
+* A Device Type
+  * A Device Type requires a Manufacturer
+* A Site
+
+Looking at the list above, there are four objects in Nautobot that must be present prior to creating a related Device.
+The following sections will guide you through how to create each object type.
+
+## Creating a Device Role
+
+To create a new Device, you will need an existing Device Role or need to create a new Device Role instance.
+
+To create a new Device Role:
+
+1. Click on **Devices** in the top navigation menu
+2. Find **Device Roles** on the drop-down menu
+3. Select `+`
+4. In the `Add a new device role` form, populate a Name
+    * The Slug will auto-populate based on the Name, but can be manually overwritten
+5. Click on Create    
+
+![](../images/getting-started-nautobot-ui/3-create%20role.png)
+
+## Creating a Manufacturer
+
+To create a new Device Type, you will need an existing Manufacturer or need to create a new Manufacturer instance.
+
+To create a new Manufacturer:
+
+1. Click on **Devices** in the top navigation menu
+2. Find **Manufacturers** on the drop-down
+3. Select `+`
+4. In the `Add a new manufacturer` form, populate the Name
+     * The Slug will auto-populate based on the Name, but can be manually overwritten
+5. Click on **Create**
+
+![](../images/getting-started-nautobot-ui/2-create%20manufacturer.png)
+
+## Create a Device Type
+
+To create a new Device, you will need an existing Device Type or need to create a new Device Type instance.
+
+To create a new Device Type:
+
+1. Click on **Devices** in the top navigation menu
+2. Find **Device Types**
+3. Select `+` 
+   
+Once on the `Add a new device type` form:
+
+4. Select the Manufacturer from the drop-down selector
+5. Select the Model from the drop-down selector
+6. Click on **Create**
```

Comment:
Unfortunately this break in the numeric list is causing them to render as a separate list. This is super annoying because I like what you're doing here. Consider the very first example under "Create a Device Type":

![Cursor_and_Creating_Devices_-_Nautobot_Documentation](https://user-images.githubusercontent.com/138052/122483658-f0a7de80-cf87-11eb-8e0a-e0920a1d950a.png)

Observe that now the list is broken up into two separate lists of 1, 2, 3, making the helpful numbering in your annotated screenshots confusing.

For this and all the other lists you've broken up, I'd suggest to just keep them bulleted numerically and used nested bullets. I hope that it can be done without having to replace the screenshots!